develop/integration/rdk-b-22q3/meta-rdk/recipes-containers/dsm/dsm.bb

```diff
@@ -0,0 +1,41 @@
+# If not stated otherwise in this file or this component's license file the
+# following copyright and licenses apply:
+#
+# Copyright 2022 Consult Red
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SUMMARY = "Downloadable Software Modules"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=50e2d278b66b3b7b20bc165c146a2b58"
+
+SRC_URI = "git://git@gitlab.consult.red/dave.chapman/rdk-smart-router.git;protocol=ssh;branch=main"
```

Comment:
Same comment as librbusprovider.bb
i.e. To confirm that SRC_URI is public, and presumably apache-licensed.

Reply:
This should not have been included in the pull request. We now have all recipes as pending merges into existing meta-layers. The recipes were removed by a previous pull request which I merged. We're pushing some improvements from our local copy of DSM to the hosted on here. Unfortunately I wasn't clear in stating it should only be the code under the src directory that should be pushed. Will either close this pull request, or fix so its only code updates, and there are no more recipes here ... as it should be.